src/Parameters.cpp

```diff
@@ -68,7 +69,8 @@ void Parameters::setup_default_parameters() {
     //cfg_num_threads = std::max(1, std::min(num_cpus, MAX_CPUS));
     cfg_num_threads = 2;
 
-    cfg_max_playouts = 800;
+    cfg_max_playouts = std::numeric_limits<decltype(cfg_max_playouts)>::max() / 2;
```

Comment:
Should we make `std::numeric_limits<decltype(cfg_max_playouts)>::max() / 2;` a constexpr somewhere since it's use so frequently?

Reply:
Added to Parameters.